src/partials/hero.html

```diff
@@ -0,0 +1,31 @@
+<!-- Hero section -->
+<section class="hero">
+  <div class="container">  <!-- tu position: relative; i elementy rozmieszczone za pomocą position: absolute; -->
+    <h1 class="hero__header">ICE CREAM MADE WITH PASSION</h1>
+    <ul class="hero__buttons">
+      <li class="hero__button">
+        <a href="#" class="button--header-left">Products</a>
+      </li>
+      <li class="hero__button">
+        <a href="#" class="button--header-right">How it's made?</a>
+      </li>
+    </ul>
+    <img src="./images/hero/hero-milk.png" alt="Bio milk" class="hero__image--milk" />
+    <div class="hero__paragraph">
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et</p>
+        <button type="button" class="hero__button--arrow"><!--grafikę strzałki umieścic w ::after--></button>
+    </div>
+    <img src="./images/hero/hero-icecream.png" alt="Tasty ice cream" class="hero__image--icecream">
+    <ul class="hero__counters">
+        <li class="hero__counter">
+            <span class="hero__counter--upper">16</span>
+            <span class="hero__counter--lower">cafes</span></li>
```

Comment:
Zmienił bym klasę na hero__place-name lub hero__spot-name, gdzie place-name lub spot-name to element sekcji nadrzędnej hero.